src/vpscrom/SS316ROMPrediction.C

```diff
@@ -0,0 +1,207 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "SS316ROMPrediction.h"
+
+SS316ROMPrediction::SS316ROMPrediction(){};
+
+void
+SS316ROMPrediction::computeROMPredictions(const double & dt,
```

Comment:
Was this file from LANL originally? I received similar functions that I used as a starting point

Reply:
No, this was my work that I shared with Arul and PA (Kairos)via my blackbear branch when they met with me in July

Reply:
From LANL I received a FORTRAN standalone version of the ROM and later a cpp version (from Aaron). I've tried to denote with comments the LANL standalone cpp version of the code

Reply:
Gotcha! Just wanted to make sure you got fair credit for all your work.